streams/examples/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -118,9 +151,13 @@ public static void main(String[] args) throws Exception {
                 });
 
         // write to the result topic
-        regionCount.to("streams-pageviewstats-output", new JsonPOJOSerializer<>(), new JsonPOJOSerializer<>());
+        regionCount.to("streams-pageviewstats-typed-output", wPageViewByRegionSerializer, regionCountSerializer);
```

Comment:
To be consistent, we should also change the job ID (further above in the file) to say "typed":

```java
props.put(StreamsConfig.JOB_ID_CONFIG, "streams-pageview-typed");
```